content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1057,6 +1057,19 @@ Used on: Pod
 The Job controller in the kube-controller-manager sets this as a label and annotation for Pods
 created with Indexed [completion mode](/docs/concepts/workloads/controllers/job/#completion-mode).
 
+### batch.kubernetes.io/cronjob-scheduled-timestamp
+
+Type: Annotation
+
+Example: `batch.kubernetes.io/cronjob-scheduled-timestamp: "2016-05-19T03:00:00-07:00"`
+
+Used on: Jobs and Pods controlled by CronJobs
+
+This annotation is used to record the original (expected) creation timestamp for a Job,
+when that Job is part of a CronJob.
+The control plane sets the value to that timestamp in RFC3339 format. If the Job belongs to a CronJob
+with a timezone specified, then the timestamp is in that timezone. Otherwise, the timestamp is in controller-manager's local time.
+
 Note the [PodIndexLabel](content/en/docs/reference/command-line-tools-reference/feature-gates.md) feature gate must be enabled
 for this to be added as a pod **label**, otherwise it will just be an annotation.
```

Comment:
Is this true of ` batch.kubernetes.io/cronjob-scheduled-timestamp`? I think this text needs to move.

Reply:
This is not related to the `batch.kubernetes.io/cronjob-scheduled-timestamp` annotation, but the  'kubectl.kubernetes.io/default-logs-container (deprecated)' one.

Reply:
Have a look at https://deploy-preview-41857--kubernetes-io-vnext-staging.netlify.app/docs/reference/labels-annotations-taints/#batch-kubernetes-io-cronjob-scheduled-timestamp

The last paragraph belongs to https://deploy-preview-41857--kubernetes-io-vnext-staging.netlify.app/docs/reference/labels-annotations-taints/#batch-kubernetes-io-job-completion-index - and you should move these 2 lines to achieve that.

Reply:
Got it. Done.